spec/src/main/asciidoc/overview.asciidoc

```diff
@@ -79,12 +79,52 @@ Instances of `ManagedExecutor` and `ThreadContext` can be injected into CDI bean
     unmanagedCompletionStage.thenApply(threadContext.contextualFunction(function3));
 ----
 
-The container provides default instances of `ManagedExecutor` and `ThreadContext`, as injected above, which are shared within an application with all other injection points for `ManagedExecutor` and `ThreadContext` that do not specify a qualifier. Their configuration is equivalent to `ManagedExecutor.builder().build()` and `ThreadContext.builder().build()`.
+The container allows to inject default instances of `ManagedExecutor` and `ThreadContext` which do not leverave either `@NamedInstance` or `ManagedExecutorConfig`/`ThreadContextConfig` annotations, as shown above.
+These instances are not shared and container creates a new instance for each injection point of `ManagedExecutor` or `ThreadContext`.
+The configuration of these instances is equal to default values present in `@ManagedExecutorConfig` and `ThreadContextConfig`.
 
 === Injection of Configured Instances
 
-TODO write this section after discussion about the Config annotations is resolved.
+The `@ManagedExecutorConfig` and `@ThreadContextConfig` provide the ability to configure the injection points with desired capabilities.
```

Comment:
The `@ManagedExecutorConfig` and `@ThreadContextConfig` annotations provide the ability to configure the injection points with desired capabilities.